src/app/src/components/ClaimedFacilitiesDetails.jsx

```diff
@@ -1,9 +1,543 @@
+                    disabled={updating}
+                />
+                <Typography
+                    variant="headline"
+                    style={claimedFacilitiesDetailsStyles.headingStyles}
+                >
+                    Office information
+                    <span
+                        style={
+                            claimedFacilitiesDetailsStyles.switchSectionStyles
+                        }
+                    >
+                        <Switch
+                            color="primary"
+                            onChange={updateOfficeVisibility}
+                            checked={data.office_info_publicly_visible}
+                        />
+                        Publicly visible
+                    </span>
+                </Typography>
+                <InputSection
+                    label="Office name"
+                    value={data.office_official_name}
+                    onChange={updateOfficeName}
+                    disabled={updating}
+                />
+                <InputSection
+                    label="Address"
+                    value={data.office_address}
+                    onChange={updateOfficeAddress}
+                    disabled={updating}
+                />
+                <InputSection
+                    label="Country"
+                    value={data.office_country_code}
+                    onChange={updateOfficeCountry}
+                    disabled={updating}
+                    isSelect
+                    selectOptions={countryOptions}
+                />
+                <InputSection
+                    label="Phone number"
+                    value={data.office_phone_number}
+                    onChange={updateOfficePhone}
+                    disabled={updating}
+                />
+                {errorUpdating && (
+                    <div style={claimedFacilitiesDetailsStyles.errorStyles}>
+                        <Typography variant="body1">
+                            <span style={{ color: 'red' }}>
+                                The following errors prevented updating the
+                                facility claim:
+                            </span>
+                        </Typography>
+                        <ul>
+                            {errorUpdating.map(err => (
+                                <li key={err}>
+                                    <span style={{ color: 'red' }}>{err}</span>
+                                </li>
+                            ))}
+                        </ul>
+                    </div>
+                )}
+                <div style={claimedFacilitiesDetailsStyles.controlStyles}>
+                    <Button
+                        onClick={submitUpdate}
+                        variant="contained"
+                        color="primary"
+                        disabled={updating}
+                    >
+                        Submit
```

Comment:
Consider "Save" or "Save Changes" instead of "Submit"

Also consider showing some "toast" or similar if there is a 200 response. Right now there is no indication that the save succeeded.

Reply:
"Save" is in 86904d276ce2250c4697174658b0f20ef45dc6ae

toast is in cd8cc08bf7b3a8a116313adf68a17112df28c48c